packages/components/src/components/table/table-row/table-row-styles.ts

```diff
@@ -6,17 +6,30 @@ import {
   hostHiddenStyles,
   hoverMediaQuery,
 } from '../../../styles';
+import { borderRadiusSmall } from '@porsche-design-system/utilities-v2';
+import type { Theme } from '../../../types';
 
-export const getComponentCss = (): string => {
+export const offset = '-2px';
+export const getComponentCss = (theme: Theme): string => {
   return getCss({
     '@global': {
       ':host': addImportantToEachRule({
+        position: 'relative',
         display: 'table-row',
         ...hostHiddenStyles,
         ...hoverMediaQuery({
-          transition: getTransition('background-color'),
-          '&(:hover)': {
-            backgroundColor: getThemedColors('light').backgroundSurfaceColor,
+          '&::before': {
```

Comment:
Would have been cool to achieve this without a pseudo element.

Reply:
@denyo Yes, it would have been cool.. I was trying to achieve this, but:

1) it's not possible to set border radius to table row, so we can't apply the hover style to :host
2) only table cells can be the children of table row, so we can't create any additional element under :host